pkg/providers/docker/testdata/cluster_main.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: anywhere.eks.amazonaws.com/v1alpha1
+kind: Cluster
+metadata:
+  name: test
+  namespace: test-namespace
+spec:
+  controlPlaneConfiguration:
+    count: 3
+  kubernetesVersion: "1.19"
```

Comment:
1.19 has been deprecated at the moment, can we use 1.22/1.23 instead?

Reply:
Fixed. Using 1.23 insead